Automattic/simplenote-electron
# Add WP Sign In

This PR adds support for signing to Simplenote via your WordPress.com account. It supports creating a new Simplenote account if one didn't exist yet, or alternatively it will sign the user to an existing account.

![wp-sign-in](https://user-images.githubusercontent.com/789137/40559834-eb66e378-600c-11e8-89d6-8ed6ac7a3af5.gif)

**To Test**
* From the login screen tap 'Sign in with WordPress.com'
* Authorize a WordPress.com account
* 3 things should happen:
 ** If a Simplenote account already exists for the account, it should be signed in successfully.
 ** If a Simplenote account didn't exist for the account, a new one should be created and you'll see the welcome note.
 ** If the WordPress.com account hasn't been verified by email, you should get an error message.

We're storing the WP.com token in the state for future integrations. If a user signs out, it will be deleted.

_Note: The reviewer will need some new keys to add to their config file. Ping me on slack to get those._
